did-tezos/Cargo.toml

```diff
@@ -8,10 +8,17 @@ edition = "2018"
 secp256k1 = ["libsecp256k1", "ssi/secp256k1", "ssi/rand"]
 
 [dependencies]
-ssi = { path = "../", default-features = false }
-chrono = { version = "0.4", features = ["serde"] }
+ssi = { path = "../" }
+# This file needs to be edited if the user want to use ed25519-dalek instead of ring.
+# The UX should improve in the future: https://github.com/rust-lang/cargo/issues/3494
+# ssi = { path = "../", default-features = false, features = ["ed25519-dalek", "rand", "sha2"] }
+chrono = { version = "0.4" }
 tokio = { version = "1.0", features = ["macros", "rt"] }
+reqwest = { version = "0.11", features = ["json", "default-tls"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 async-trait = "0.1"
 libsecp256k1 = { version = "0.3", optional = true }
+anyhow = "1.0.33"
+json-patch = "0.2.6"
+tezedge_client = { git = "https://github.com/binier/tezedge-client", rev = "672e61297f0ba7d1d94168a2588b9b27d76c1fc4", package = "lib" }
```

Comment:
I think we already have the functionality that `tezedge_client` is being used for here, except for P-256:
- `bs58` is a dependency of `ssi`, used for base58check [encoding for did:tz](https://github.com/spruceid/ssi/blob/main/src/blakesig.rs#L23-L51)
- JWK struct represents public and private keys for ed25519 and secp256k1

Reply:
This is true. I added it thinking we will use it in the long term, at least for the DID Manager operations. But we can leave it out for the resolution, I don't know, I think I would prefer to use it in libssi once it handles all address types so the decoding and other operations are centralised.

Reply:
I'm okay keeping this dependency for now, but agree that simpler is better. I have tracked it here: https://github.com/spruceid/ssi/issues/112

Reply:
This was addresed in d2e7792686096fead5c2e69bc547939f791d670d